src/cell/main.c

```diff
@@ -118,6 +239,18 @@ parse_command_line(int argc, char* argv[])
 				print_banner(true, true);
 				return false;
 			}
+			else if (strcmp(argv[i], "--init") == 0) {
+				printf("Generating new project\n");
+				if (++i >= argc) goto missing_argument;
+				if (fexist(argv[i])) {
+					printf("cell: directory '%s' already exists\n", argv[i]);
```

Comment:
Hmm... maybe it's just my Git experience speaking but I would intuitively expect something like this to work:
```
mkdir new_thing
cd new_thing
cell --init .
```
but the existence check prevents that from working, right?

Reply:
It creates the given directory simply for convenience. And yes, while dircopy() doesn't check for the existence of the given directory, it does check in main.c on line 245. It would be pretty trivial to have dircopy do that as well or to have it copy the files to the current directory as you describe if no directory is given.